reflection/reflection.fbs

```diff
@@ -38,6 +38,10 @@ table Type {
                      // If base_type == Union, UnionType, or integral derived
                      // from an enum, index into "enums" below.
     fixed_length:uint16 = 0;  // Only if base_type == Array.
+    /// The size (octets) of the `base_type` field.
+    base_size:uint8 = 4; // 4 Is a common size due to pointers being that size.
+    /// The size (octets) of the `element` field, if present.
+    element_size:uint8 = 0;
```

Comment:
structs can be way bigger size than what fits in a byte

Reply:
I didn't want to duplicate data. The struct length is in `Object.bytesize`.

Reply:
I bumped them up to uint32 anyways.

I think its outside the scope of this PR to fix the struct size issue.